README.md

```diff
@@ -1 +1,57 @@
-# node-repl-lab
\ No newline at end of file
+# Running JavaScript Code in Node REPL
+
+## Objectives
+
+1. Use REPL
+1. Execute code in REPL
+1. Exit REPL
+
+## Introduction
+
+Understanding global and scoping will give you a good foundation for future development.
+
+## Instructions
+
+1. Enter REPL
+1. Write Node code to perform math by creating a sum function
+1. Call the sum function to add numbers
+2. Find out what the day of the week was the day of the Declaration of Independence (July 4th, 1776)
+1. Exit REPL
+
+
+### Extra Info
+
+
+
+* [Arguments object on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments)
+* [Date on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)
+
+## Solution
+
+A sum function for two numbers:
+
+```js
```

Comment:
we stressed that students use function declaration as opposed to function expression - maybe provide both ways of doing it